sdk/typescript/test/e2e/multisig.test.ts

```diff
@@ -899,4 +897,36 @@ describe('Multisig address creation:', () => {
 			}),
 		).toThrow(new Error('Multisig does not support duplicate public keys'));
 	});
+
+	it('multisig address with zklogin pubkey', async () => {
```

Comment:
Does this actually include a zklogin pubkey?

Reply:
/// A wrapper struct to retrofit in [enum PublicKey] for zkLogin.
/// Useful to construct [struct MultiSigPublicKey].

its probably going to implement the publickey interface in typescript. i call it the public identifier since its not a public key in the sense its not derived from a private key. but it is the public field that the zklogin address is derived from